src/main/java/org/deepsymmetry/beatlink/MediaDetails.java

```diff
@@ -131,24 +129,18 @@ private int getUTF16StringLength(byte[] source, int offset, int maxLength) {
      * @param mediaType     Media Type to Emulate
      * @param name          Name of device
      */
-    MediaDetails(SlotReference slotReference, CdjStatus.TrackType mediaType, String name) {
+    public MediaDetails(SlotReference slotReference, CdjStatus.TrackType mediaType, String name,
```

Comment:
Why is this now public? Clients of Beat Link should not need to call it, only internal code. And I don’t see anywhere we are calling it outside of the package. But I can go ahead and merge this and change that back.